test/test_coverer.py

```diff
@@ -76,6 +76,14 @@ def test_perc_overlap(self, CoverClass):
         for i in (1, 2, 3, 4):
             assert data[i] in entries[1]
 
+    def test_contains(self, CoverClass):
```

Comment:
could you add a test that shows this works in higher number of dimensions as well?